packages/testcontainers/src/wait-strategies/utils/port-check.ts

```diff
@@ -4,15 +4,21 @@ import { log } from "../../common";
 import { ContainerRuntimeClient } from "../../container-runtime";
 
 export interface PortCheck {
-  isBound(port: number): Promise<boolean>;
+  isBound(port: number | string): Promise<boolean>;
 }
 
 export class HostPortCheck implements PortCheck {
   constructor(private readonly client: ContainerRuntimeClient) {}
 
-  public isBound(port: number): Promise<boolean> {
+  public isBound(port: number | string): Promise<boolean> {
+    if (typeof port === "string" && port.toLowerCase().includes("/udp")) {
```

Comment:
It's more precise to replace `includes` with `endsWith`, same in `InternalPortCheck`

Reply:
Changed